README.md

```diff
@@ -1,355 +1,355 @@
- * [Fitness Challenge](https://play.google.com/store/apps/details?id=com.isidroid.fitchallenge)
- * [Crunch (ICE)](https://play.google.com/store/apps/details?id=com.figsandolives.ice.free)
- * [Filmy - Your Movie Guide](https://play.google.com/store/apps/details?id=tech.salroid.filmy)
- * [HEBF Optimizer ▪ Root](https://play.google.com/store/apps/details?id=com.androidvip.hebf)
- * [Wifi Captive Login](https://play.google.com/store/apps/details?id=com.anantharam.wificaptivelogin)
- * [IIFYM](https://play.google.com/store/apps/details?id=com.javierd.iifym)
- * [Ampwifi Winamp Remote](https://play.google.com/store/apps/details?id=com.blitterhead.ampwifi)
- * [AaiKya: Leave Tracker](https://play.google.com/store/apps/details?id=com.ranveeraggarwal.letrack)
- * [Angopapo - People around you](https://play.google.com/store/apps/details?id=com.msingapro.angopapofb)
- * [Hugetwit](https://play.google.com/store/apps/details?id=com.halilibo.hugetwit)
- * [Wake Me Up (Mumbai Railway)](https://play.google.com/store/apps/details?id=com.catacomblabs.wakemeup)
- * [SelfMote - Wireless Remote app](https://play.google.com/store/apps/details?id=com.dmicse.selfmote.free)
- * [Boo Music Player](https://play.google.com/store/apps/details?id=cdn.BooPlayer)
- * [BeatPrompter](https://play.google.com/store/apps/details?id=com.stevenfrew.beatprompter)
- * [Orario Treni Trenitalia](https://play.google.com/store/apps/details?id=com.jaus.albertogiunta.justintrain_oraritreni)
- * [Tipsy for Gardaland](https://play.google.com/store/apps/details?id=com.MonkeyLab.MyGardaland&hl=it)
+[![](https://jitpack.io/v/paolorotolo/AppIntro.svg)](https://jitpack.io/#paolorotolo/appintro)
+[![Android Arsenal](https://img.shields.io/badge/Android%20Arsenal-AppIntro-green.svg?style=flat)](https://android-arsenal.com/details/1/1939)
+[![Android Gems](http://www.android-gems.com/badge/PaoloRotolo/AppIntro.svg?branch=master)](http://www.android-gems.com/lib/PaoloRotolo/AppIntro) [![Join the chat at https://gitter.im/AppIntro/Lobby](https://badges.gitter.im/AppIntro/Lobby.svg)](https://gitter.im/AppIntro/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+# AppIntro
+AppIntro is an Android Library that helps you make a **cool intro** for your app, like the ones in Google apps.
+
+<img src="https://github.com/PaoloRotolo/AppIntro/blob/master/art/intro.png" width="300"> <img src="https://github.com/PaoloRotolo/AppIntro/blob/master/art/layout2.png" width="300">
+
+*Watch the [demo video](https://www.youtube.com/watch?v=-KgAAbZz248) on YouTube*
+
+## Usage
+
+### Basic usage
+
+1. **Add the JitPack repository to your build file**
+
+ Add it in your root build.gradle at the end of repositories:
+```gradle
+	allprojects {
+		repositories {
+			...
+			maven { url 'https://jitpack.io' }
+		}
+	}
+```
+
+2. **Add the dependency**
+```gradle
+	dependencies {
+	        implementation 'com.github.paolorotolo:appintro:v5.1.0'
```

Comment:
Ok, you probably hate me right now, but just one more thing :D
Can you keep the readme version to 5.0.1 (current release?). I want to add some other things before releasing 5.1.0 to the public.

Reply:
@paolorotolo no problem, will do tomorrow ^^
Glad to be helpful :smile:

Reply:
Changed on commit 11f3e06 @paolorotolo